news/_posts/2019-10-14-prestashop-1.7-modules-latest-releases-and-updates.md

```diff
@@ -0,0 +1,95 @@
+img
+
+- Let users upvote or downvote a review or report an abuse if a comment is not appropriate
+- Let merchants moderate reviews they receive on their products  
+img
+
+- Rich snippets integration
+
+
+### Faceted Search module (current version: 3.3.0)
+
+A huge refactoring has been made for version 3 and a lot of unit tests have been added to prevent future regressions. Many improvements have been added:  
+
+- Indexation and cache management have been improved for better performances
+- We added price and weight ranges sliders that are also working on mobile ([#9992](https://github.com/PrestaShop/PrestaShop/issues/9992))
+- Filters which do not match any products are now hidden ([#14655](https://github.com/PrestaShop/PrestaShop/issues/14655))
+
+A lot of bugs have been solved, including:
+
+- You can now limit filter results ([#9813](https://github.com/PrestaShop/PrestaShop/issues/9813))
+- The number of matching products after selecting filters is now working ([#14734](https://github.com/PrestaShop/PrestaShop/issues/14734))
+- Disabling the number of matching products is now working ([#9949](https://github.com/PrestaShop/PrestaShop/issues/9949))
+- Long feature titles are truncated to keep them on one line ([#9978](https://github.com/PrestaShop/PrestaShop/issues/9978))
+- Products are correctly sorted by position when you don’t show products from subcategories ([#15062](https://github.com/PrestaShop/PrestaShop/issues/15062))
+- Products are correctly sorted by price even when you have discounts ([#15223](https://github.com/PrestaShop/PrestaShop/issues/15223) & [#10010](https://github.com/PrestaShop/PrestaShop/issues/10010))
+- You can choose in which order the filters appear (#9880)
+- “Filter by” blocks are not displayed if no filters are available ([#9732](https://github.com/PrestaShop/PrestaShop/issues/9732) & [#9736](https://github.com/PrestaShop/PrestaShop/issues/9736))
+
+### Customer reassurance module (current version: 4.1.1)
+
+A major update on the module has been released last week bringing new features and design both on front office and back office. With this new version, merchants will be able to: 
+
+- Display their elements as desired on their homepage, either on the header or footer, but also on product and checkout pages  
+img
+
+- Create as many elements as desired, with the ability to add available or custom icons, title and description and choose colors  
+img
+
+- Enable or disable any of them and easily reorder them through drag and drop feature  
+img
+
+### Other bug fixes on modules 
+
+- Email subscription module (v2.5.0) - Fix on email registration issue ([#15425](https://github.com/PrestaShop/PrestaShop/issues/15425))
+- Contact form (v4.1.1) - Add a check to verify if the customer is the order’s owner
+- Shopping cart module (v2.0.2) - Fix related to add to cart from quickview issue ([#11631](https://github.com/PrestaShop/PrestaShop/issues/11631))
```

Comment:
There is a few nice work in progress on `dev` branch for this module (https://github.com/PrestaShop/ps_shoppingcart/commits/dev) however it has not been released yet. Shall we mention it, here or maybe in the section below ?

Reply:
this will make some content for a future article on module news ;-)